cmd/cli/commands/deals.go

```diff
@@ -143,8 +143,8 @@ var dealOpenCmd = &cobra.Command{
 }
 
 var dealQuickBuyCmd = &cobra.Command{
-	Use:    "quick-buy <ask_id>",
-	Short:  "Copy given ASK order with BID type and open a deal with this orders",
+	Use:    "quick-buy <ask_id> [duration]",
```

Comment:
I'd prefer to implement this as the cobra flag with Duration type: less parsing, less checking for an args count, less description into a command (will be moved to the flag's help message).

Reply:
Thanks to our clear api we need to distinguish between 0 duration and it's absence. It would be quite dirty using the flag. Besides I find `sonmcli quickbuy 42 1h` more friendly than `sonmcli quickbuy --duration=1h 42`

Reply:
But if the user agrees with proposed ASK duration it not necessary to explicitly set deal duration, we can handle it on the Node, like the change request API do.